GCTC-NTGC/gc-digital-talent
# Align PostgreSQL versions

Resolves #3827.

- Adds `pgcrypto` extension that is required for the `gen_random_uuid()` function called in several migration files and is [not available before PostgreSQL 13](https://pgpedia.info/g/gen_random_uuid-function.html).

### Steps to test
1. `docker-compose down`
2. `docker-compose up --detach --build`
3. `docker-compose run --rm maintenance bash setup.sh`
4. `docker-compose run --rm maintenance bash refresh_all.sh`
5. `docker-compose exec -w /var/www/html/api php sh -c "php artisan migrate"`
6. `docker-compose exec -w /var/www/html/api php sh -c "php artisan db:seed"`
7. Verify there are no build errors
